src/main/java/com/github/exadmin/sourcesscanner/exclude/ExcludeFileModel.java

```diff
@@ -42,11 +42,9 @@ public void doSortBeforeSaving() {
     }
 
     public boolean isPathFullyIgnored(String relFileOrDirNameHash) {
-        for (ExcludeSignatureItem next : signatures) {
-            if (next.getFileHash().equals(relFileOrDirNameHash) && next.getTextHash().equals(SKIP_FULL_FILE_HASH)) return true;
-        }
+    return signatures.stream()
+        .anyMatch(item -> item.getFileHash().equals(relFileOrDirNameHash) 
+            && item.getTextHash().equals(SKIP_FULL_FILE_HASH));
```

Comment:
Add a space after the equals operator for consistent formatting.

```suggestion
            && item.getTextHash().equals(SKIP_FULL_FILE_HASH));
```

#deepseek-review:inline